### DISCUSSION TITLE
How to show tooltips for elements?

### DISCUSSION THREAD

**Discussion by reshin-pulapaddy** [clientIO/joint#2445] [Q&A]
### Introduction

I am using the React KitchenSink application
How to show a tooltip for elements when we hover over an element?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment]
There are many ways and one of them is:

```js
new joint.ui.Tooltip({
    target: '[data-tooltip]',
    direction: 'auto',
    padding: 10
});

element1.attr(['root', 'dataTooltip'], 'My Tooltip');
```
For more information, please see the [docs](https://resources.jointjs.com/docs/rappid/v3.7/ui.Tooltip.html#ui.Tooltip
).

**Reply by reshin-pulapaddy** [reply]
Thank you @kumilingus
